src/main/scala/it/pagopa/interop/backendforfrontend/service/AgreementProcessService.scala

```diff
@@ -45,4 +45,12 @@ trait AgreementProcessService {
   def removeConsumerDocument(agreementId: UUID, documentId: UUID)(implicit
     contexts: Seq[(String, String)]
   ): Future[Unit]
+
+  def getAgreementEServices(
+    eServiceName: Option[String],
+    producersIds: Seq[UUID] = Seq.empty,
+    consumersIds: Seq[UUID] = Seq.empty,
+    limit: Int,
+    offset: Int = 0
```

Comment:
let's avoid default values if not strictly required
```suggestion
    offset: Int
```